### ISSUE TITLE
Feat: allow to identify some spaces by a color

### ISSUE THREAD

**Issue by lebouquetin** [tracim/tracim#3775]
## Feature description and goals

Color is an immediate tool for identification of items. Scrolling in the side bar for all spaces may be inefficient when all you have to identify the space you are looking for is the label of the space.

Defining a space color would help to fast identify a given space.

Example of what is expected:

![image](https://user-images.githubusercontent.com/907161/97732259-93cde580-1ad6-11eb-939f-f1d7ed39fbac.png)

Note : color may be associated to space for all users or per user (so everybody can define own colors... to be discussed)

**Comment by raphj** [comment]
I think this should be a per-user configuration, to add to the user's space screen maybe.

Alternatively, it could be a colored circle at the right of the sidebar (a bit like the circle in an unread notification)

**Comment by PhilippeAccorsi** [comment]
I started with the possibility to use colours so that I could easily find a space or spaces when I have a lot to scroll.

This avoids having to read the titles to find the spaces that are used the most.

**Comment by grignards** [comment]
I agree that having a functionality to easily find spaces is a plus. Having colors in the sidebar like proposed in this issue is one possible solution but I can see others: having favorites, allowing to filter in the sidebar/… Should we list the solutions we see and evaluate them in order to choose one which seems the more appropriate?

**Comment by gbonaspetti** [comment]
I am a little afraid of this feature:
- One color for each space can create a carnival of colors and will do almost nothing to help identify spaces. Giving the user the ability (without obligation) to add colors opens up the possibility of good or bad (like put one color for space) use of this feature.
- Adding colors distracts attention from "call to action" and other things that we chose, guides the user's look to the sidebar instead of guiding to the content. (I advise to look, particularly the image of item 14 of this site 
 https://www.orbitmedia.com/blog/web-design-tips/)
